javascript/javascript1/week4/VoiceAssistant.js

```diff
@@ -0,0 +1,180 @@
+        } 
+        else 
+        {
+            commandHistory.push(command.toLowerCase()) ;
+            console.log(`nice to meet you ${curUser}`) ;
+        }
+    } 
+// If asking for name, return current user
+    else if(command.toLowerCase().startsWith("what is my name")) 
+    { 
+        if (curUser.length) 
+        {
+            console.log(`Your name is: ${curUser}`);
+        } 
+        else 
+        {
+            console.log(`Sorry we don't recognize you`);
+        }
+    } 
+//Add todo list items
+    else if(command.toLowerCase().startsWith("add") &&
+    command.toLowerCase().endsWith("to my todo")) 
+    {
+
+        const task = command.replace("Add ", "").replace(" to my todo", "");
+        console.log("Added " + task + " to todo");
+        todos.push(task);
+    }
+//Remove todo list items 
+    else if(command.toLowerCase().startsWith("remove") &&
+    command.toLowerCase().endsWith("from my todo")) 
+    {
+        const task = command.replace("Remove ", "").replace(" from my todo", "");
+
+        let removed = false;
+        for(item in todos) 
+        {
+            if(todos[item] == task) 
+            {
+                todos.splice(item, 1);
+                console.log(`Removed ${task} to todo`);
+                removed = true;
+                break;
+            }
+        }
+
+        if (!removed) 
+        {
+            console.log(`Could not find ${task} in todo`);
+        }
+    }
+//Print todo list items 
+    else if(command.toLowerCase().includes("what is on my todo")) 
+    {
+        console.log("\nItems in Todo");
+        for(item in todos) 
+        {
+            console.log(" - " + todos[item]);
+        }
+    }
+//Simple arithmetic calculations
+    else if(command.toLowerCase().startsWith("what is"))
+    {
+        const calculateCommand = command.split(" ");
+        const firstNumber =  parseInt(calculateCommand[2]);
+        const secondNumber =  parseInt(calculateCommand[4]);
+        let result = 0;
+        if(calculateCommand[3] ===  '+')
+        {
+            result=firstNumber + firstNumber;
+            console.log(result);
+           
+        }
+       if(calculateCommand[3] === '-')
+        {
+            result=firstNumber - firstNumber;
+            console.log(result);
+        }
+        if(calculateCommand[3] === '*')
+        {
+            result=firstNumber * firstNumber;
+            console.log(result);
+        }
+        if(calculateCommand[3] === '/')
+        {
+            result=firstNumber / firstNumber;
+            console.log(result);
+        }
+        if(calculateCommand[3] === '%')
+        {
+            result=firstNumber % firstNumber;
+            console.log(result);
+        }
+        
+    }
+//Print current day in a human readable format
+    else if (command.includes("what day is today"))
+    {
+        const today = new Date().toDateString();
+        arrDate = today.split(" ");
```

Comment:
If you want to use this as a variable you have to assign it to either `let` or `const`.

Reply:
There's a trick called "array destructuring", which can be used to assign array items easily:
```js
const today = new Date().toDateString();        
const [ weekday, month, day, year ] = today.split(" "); // this way, we assign each element in the array to a const.
console.log(`${day}. of ${month} ${year}`);
```